mobile/src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -406,7 +410,13 @@ const Counter = <T,>({
 
   if (!isSearchOpened && fungibilityFilter === 'all') {
     return (
-      <View style={[a.p_lg, {justifyContent: 'center', flexDirection: 'row'}]}>
+      <View
+        style={[
+          a.px_lg,
+          a.py_sm,
+          {justifyContent: 'center', flexDirection: 'row'},
+        ]}
+      >
         <Text style={[{color: p.primary_600}, a.body_2_md_regular]}>
```

Comment:
for texts, try to figure out the right ta.text_<>... i think here is primary_medium

Reply:
done